### PR TITLE
Cleanup content for passover - use the SDK instead of having duplicate files

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22938

## Depends on:
https://github.com/demisto/demisto-sdk/pull/301
https://github.com/demisto/content-test-conf/pull/777

## Description
Deleted the duplicate files in the repo and the SDK. Also made adjustments so files can import from the SDK instead of the old files.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

**Comment by Itay4** [comment]
has sameach

File: Tests/scripts/configure_tests.py [hunk 1]
```diff
@@ -14,14 +14,35 @@
...
+    PACKS_DIR, SCRIPTS_DIR)
+INTEGRATION_REGEXES = [
+    INTEGRATION_REGEX,
+    BETA_INTEGRATION_REGEX,
+    PACKS_INTEGRATION_REGEX
+]
+INCIDENT_FIELD_REGEXES = [
+    INCIDENT_FIELD_REGEX,
+    PACKS_INCIDENT_FIELDS_REGEX
+]
```
Comment (yaakovi) [inline Tests/scripts/configure_tests.py]: why aren't those defined in the SDK?

File: .circleci/config.yml [hunk 2]
```diff
@@ -78,17 +78,10 @@ jobs:
                 exit 0
             fi
 
-            python -m pytest ./Tests/scripts/hook_validations/tests/ -v
-            # after moving to all to python3, run the tests by folder
-            # python -m pytest ./Tests/scripts/infrastructure_tests/ -v
-
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/constants_regex_test.py -v
             python -m pytest ./Tests/scripts/infrastructure_tests/mock_unit_test.py -v
```
[the remark this answers is not in the document]
Comment (liorblob) [reply .circleci/config.yml]: I did not, there was no need

File: Tests/scripts/configure_tests.py (hunk 1 above)
Comment (liorblob) [reply Tests/scripts/configure_tests.py]: they're only used here, we can move them after dean's changes

File: .circleci/config.yml [hunk 3]
```diff
@@ -78,17 +78,10 @@ jobs:
...
             python3 -m pytest ./Tests/scripts/infrastructure_tests/release_notes_test.py -v
-            python -m pytest ./Tests/scripts/infrastructure_tests/validate_files_test.py -v
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/update_id_set_test.py -v
             python3 -m pytest ./Tests/scripts/infrastructure_tests/store_packs_test.py -v
-            python -m pytest ./Tests/scripts/test_configure_tests.py -v
+            python3 -m pytest ./Tests/scripts/test_configure_tests.py -v
```
[the remark this answers is not in the document]
Comment (liorblob) [reply .circleci/config.yml]: @DeanArbel

[the remark this answers is not in the document]
Comment (DeanArbel) [reply .circleci/config.yml]: I can take care of it as part of the content build improvement PR

[the remark this answers is not in the document]
Comment (liorblob) [reply .circleci/config.yml]: is the configure_tests refactor complete?

[the remark this answers is not in the document]
Comment (DeanArbel) [reply .circleci/config.yml]: @liorblob waiting to be merged https://github.com/demisto/content/pull/6220

File: Tests/scripts/configure_tests.py (hunk 1 above)
Comment (yaakovi) [reply Tests/scripts/configure_tests.py]: weird...
no need for now.

File: .circleci/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (yaakovi) [reply .circleci/config.yml]: you moved `test_content.py` which uses it (only place)

File: .circleci/config.yml [hunk 4]
```diff
@@ -78,17 +78,10 @@ jobs:
...
-            # python -m pytest ./Tests/scripts/infrastructure_tests/ -v
-
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/constants_regex_test.py -v
-            python -m pytest ./Tests/scripts/infrastructure_tests/mock_unit_test.py -v
+            python3 -m pytest ./Tests/scripts/infrastructure_tests/mock_unit_test.py -v
             python3 -m pytest ./Tests/scripts/infrastructure_tests/release_notes_test.py -v
-            python -m pytest ./Tests/scripts/infrastructure_tests/validate_files_test.py -v
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/update_id_set_test.py -v
             python3 -m pytest ./Tests/scripts/infrastructure_tests/store_packs_test.py -v
-            python -m pytest ./Tests/scripts/infrastructure_tests/test_configure_tests.py -v
```
Comment (yaakovi) [inline .circleci/config.yml:91]: `python3 -m pytest ./Tests/scripts/infrastructure_tests/ -v`

remove all redundant test files (constants_regex_test, content_creator_test, update_id_set_test, validate_files_test)

**Pull request merged**